yarn.lock

```diff
@@ -13719,7 +13726,7 @@ source-map-support@^0.4.15:
   dependencies:
     source-map "^0.5.6"
 
-source-map-support@^0.5.0, source-map-support@^0.5.1, source-map-support@^0.5.13, source-map-support@^0.5.3, source-map-support@^0.5.6:
+source-map-support@^0.5.0, source-map-support@^0.5.1, source-map-support@^0.5.16, source-map-support@^0.5.3, source-map-support@^0.5.6:
```

Comment:
can we clean up the `package.json`s so we only ever request one version of this? (it's just getting a bit out of hand)

Reply:
Hm, I'm not sure why this is happening -- the `package.json`s all say `^0.5.16`.  No idea why it's so messy here.  Is there something I missed somehow?

Reply:
oh maybe it's our dependencies that depend on `source-map-support`! nevermind then!